pages/hire.vue

```diff
@@ -53,29 +53,39 @@ export default {
         hire: {
           items: [],
           limit: 16,
-          skip: 0,
-        },
-      },
+          skip: 0
+        }
+      }
     }
   },
   methods: {
     async loadMorePersons($state) {
       this.postList.hire.skip += this.postList.hire.limit
 
-      const hire = await this.$content('persons')
+      const hire = await this.$content("persons")
         .where({ isHireable: true })
         .limit(this.postList.hire.limit)
         .skip(this.postList.hire.skip)
         .fetch()
 
+      // Temporary for duplicated record
```

Comment:
We have to resolve this problem in a proper way, we can loop over all persons again in this approach

Reply:
Thank you, this pull request can now be closed and let the looking for new thinking